docs/meshcloud.customer.md

```diff
@@ -7,6 +7,12 @@ title:  Customer Accounts (meshCustomer)
 
 A meshCustomer is automatically created for you when you [sign up](meshcloud.profile.md) for a new account. A meshCustomer usually represents your company or department. Within a meshCustomer you manage multiple [projects](meshcloud.project.md), your users and their access rights and maintain general data like addresses and payment methods.
 
+## Customer Settings
+
+General information of a meshCustomer, like its name can be edited here. The customer identifier is also shown here, but this identifier can never be changed, as it is used as an immutable identifier of the meshCustomer for its representation in the different cloud platforms.
```

Comment:
At first glance I was a bit confused about the "like its" wording and also there are some word repetitions. I suggest:
```suggestion
General information of a meshCustomer (like its name) can be edited here. The customer identifier is also shown here, but it can never be changed, as it is used as an immutable identifier of the meshCustomer for its representation in the different cloud platforms.
```
Also the line seems a bit long to me. I know its currently disabled and there are a lot of overlong lines already there, but maybe we dont produce new ones so we can enable this markdownlint feature (I suggest 120 or 140 chars per line)

Reply:
hmm, i am not sure. VS Code breaks the line nicely at the end of the window. So i'd prefer to leave the line breaks up to the Editor. Otherwise it might look strange dependend on your resolution or window size.